lib/active_record/connection_adapters/sqlserver_adapter.rb

```diff
@@ -1,6 +1,7 @@
 require 'base64'
 require 'active_record'
 require 'arel_sqlserver'
+require 'arel/visitors/sqlserver'
```

Comment:
Are we sure on this? I never found that we wanted to couple to the core SQL Server visitor before. Why the require?